V2rayNG/app/src/main/kotlin/com/v2ray/ang/ui/MainActivity.kt

```diff
@@ -411,6 +412,9 @@ class MainActivity : BaseActivity(), NavigationView.OnNavigationItemSelectedList
                         URL(url).readText()
                     } catch (e: Exception) {
                         e.printStackTrace()
+                        Looper.prepare()
+                        toast("\"" + it.second.remarks + "\" " + getString(R.string.toast_failure))
+                        Looper.loop()
```

Comment:
I recommend not to call looper method in coroutine. Please use launch(main) to post messages on main thread.

Reply:
Thank you,it's done